test/run_regression_tests.sh

```diff
@@ -104,8 +104,11 @@ function run_tests() {
         suites+=($SCRIPTDIR/falco_tests_package.yaml)
     fi
     
+    mkdir -p ${OPT_BUIL_DIR}/integration-tests-xunit
```

Comment:
```suggestion
    mkdir -p ${OPT_BUILD_DIR}/integration-tests-xunit
```

You probably mean `OPT_BUILD_DIR` here.

Reply:
Thank you!